buildenv/jenkins/common/build.groovy

```diff
@@ -585,26 +585,75 @@ def cleanWorkspace (KEEP_WORKSPACE) {
     }
 }
 
+def clean_docker_containers() {
+    println("Listing docker containers to attempt removal")
+    sh "docker ps -a"
+    def containers = sh (script: "docker ps -a --format \"{{.ID}}\"", returnStdout: true).trim()
+    if (containers) {
+        println("Stop all docker containers")
+        sh "docker ps -a --format \"{{.ID}}\" | xargs docker stop"
+        println("Remove all docker containers")
+        sh "docker ps -a --format \"{{.ID}}\" | xargs docker rm --force"
+    }
+}
```

Comment:
How does this differ from what we've used in the past for other jobs?
```
docker system prune -af
```
Are we trying to cleanup everything or just containers/images for this job?

Reply:
This just covers stopping and removing any running containers, which is really not that necessary as the job does that on its own if it completes cleanly.  I'm not sure we want to add a `system prune` to each job as that would delete and force a fresh download of an image each time when its rarely going to be required.